src/Appwrite/Transformation/Adapter/Preview.php

```diff
@@ -47,7 +47,7 @@ public function transform(): void
                 position: fixed;
                 right: 16px;
                 bottom: 16px;
-                z-index: 1;
+                z-index: calc(infinity);
```

Comment:
!important too, maybe? 🤔

Reply:
try to have custom css on your site, and do
```
#appwrite-preview {
  z-index: -1 !important;
  }
```

By adding important to ours, we prevent others from changing it easily. We should probably do same for all params

Reply:
that I understood but not quite the point.
This isn't a general usecase and something common. I can still add it but someone can also hide the whole banner with JS as well 😉.